### PR TITLE
docs: update references in docs file

### PR DESCRIPTION
**Pull request by aanchal88**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ There are two types of content in the documentation:
 API docs are generated directly from the source code.
 The source code is contained in TypeScript files, located in the `angular/packages` folder.
 Each API item may have a preceding comment, which contains JSDoc style tags and content.
-The content is written in markdown. To generate docs, each package's files need to be explicitly included in the [packages/BUILD.bazel](../packages/BUILD.bazel) file under the `files_for_docgen` target.
+The content is written in markdown. To create docs, each package's files need to be explicitly included in the [packages/BUILD.bazel](../packages/BUILD.bazel) file under the `files_for_docgen` target.
```
Comment (crisbeto) [inline aio/README.md:62]: What's the reasoning behind this change?

Comment (aanchal88) [reply aio/README.md:62]: Hii @crisbeto, I was going through the issues and I came across the issue-46723 "docs: replace "view" term with more explicit "rendered DOM structure" or equivalent term" So to make the docs more appropriate for all the components, raised the PR

Comment (crisbeto) [reply aio/README.md:62]: It seems like we didn't agree on the terminology in #46723.

Comment (alxhub) [reply aio/README.md:62]: I'm also a little confused as #46723 relates to the term `view` whereas this PR is replacing the word `generate` in completely different contexts. I don't think the `generate` -> `create` substitution improves the legibility or quality of the docs.

**Comment by alxhub** [comment]
I'm going to close this PR as the `generate` -> `create` change is not one which we would like to make in the wording of these docs.

**Pull request closed without merging**